test/core/event_engine/work_queue/corpora/a02dfa6454c3047ba1eb04274980c636d1e56875

```diff
@@ -0,0 +1,96 @@
+actions {
```

Comment:
I'd probably drop these for now, and just check in one 'empty' file - the fuzzing backends will keep a big corpus for us, so this mostly just increases the number of bytes we need to check out to get to building.

We used to check in big corpora to github and I don't think I've seen any benefit.

Reply:
Nuked.